### PR TITLE
Added basic headings

### PR DESCRIPTION
**Pull request by MJholland**
Follow along content

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en-ca">
+<head>
+  <meta charset="utf-8">
+  <title>Web app</title>
```
Comment (scallsen) [inline index.html]: Why not use our app's name, Workspace

File: index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+</head>
+
+<body>
+
+  <h1>Web app</h1>
```
Comment (scallsen) [inline index.html]: Here as well

File: index.html (hunk 1 above)
Comment (scallsen) [inline index.html]: We should use our web app's name, Workspace

**Review by scallsen — CHANGES_REQUESTED** [review]
Some text changes to make it more clear as to the goal of our web app.

**Review by scallsen — APPROVED** [review]
Looks good, all changes completed.

**Comment by MJholland** [comment]
@thomasjbradley we complete the branch and pull requests in-class task. You were busy with groups so we couldn't show you in person.

**Pull request merged**